go-controller/pkg/ovn/secondary_layer2_network_controller.go

```diff
@@ -299,6 +299,10 @@ func (oc *SecondaryLayer2NetworkController) Start(ctx context.Context) error {
 		klog.Infof("Starting controller for secondary network %s took %v", oc.GetNetworkName(), time.Since(start))
 	}()
 
+	if err := oc.BaseNetworkController.init(); err != nil {
+		return err
```

Comment:
nit: fmt.Errorf(failed to initialize base network controller ...). I know youre following convention but i dont like that we dont enrich the errors

Reply:
ignoring for now